api4/team.go

```diff
@@ -1275,12 +1275,23 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	emailList := model.ArrayFromJSON(r.Body)
+	memberInvite := model.MemberInvite{}
```

Comment:
if you instantiate here as `&model.MemberInvite{}`, you could drop the `&` from the next line but also from function call `App.InviteNewUsersToTeamGracefully`

Reply:
ok, thanks.

Reply:
done